Adds display component for widgets with no content

Zachary-Dernar opened the pull request:
Also modifies the message found in the No Results page
to reflect the wording in the No Content component.

Zachary-Dernar committed:
Adds display component for widgets with no content

Also modifies the message found in the No Results page
to reflect the wording in the No Content component.

Zachary-Dernar committed:
Merge branch 'master' into zmdernar/no-comentions-content

maniax89 commented on views/NoContent/index.jsx:
```diff
@@ -0,0 +1,31 @@
+const NoContent = (props) => (
+  <div className="no-content--wrapper">
+    <div className="no-content--icon">
+      <Icon style={{ width: '3.7rem', height: '3.7rem', fill: '#00bfb0' }} type="info" />
```
try using css `.no-content--icon .icon` for your CSS

```
.no-content--icon .icon {
  width: 3.7rem;
  height: 3.7rem;
  fill: #00bfb0;
}
```

maniax89 commented on views/NoContent/index.jsx:
```diff
@@ -0,0 +1,31 @@
+  </div>
+);
+
+// <p className={props.query.restrictedDateRange ? 'no-content--timeframe' : 'hidden'}>
```
remove this

maniax89 reviewed (commented):
add a test for both

- Mentions And Sentiments
- Top Entities

for when <NoContent /> is shown

Zachary-Dernar committed:
Fixes suggestions and adds tests for NoContent component

Moves icon styling to css file, removes commented code,
and adds tests for both widgets that could show the
NoContent component.

Zachary-Dernar committed:
Merge branch 'zmdernar/no-comentions-content' of https://github.com/watson-developer-cloud/discovery-nodejs into zmdernar/no-comentions-content

maniax89 commented on test/views/TopEntities/test.index.js:
```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import assert from 'assert';
+import { shallow } from 'enzyme';
+//import NoContent from '../../../views/NoContent/index.jsx';
```
remove this

Zachary-Dernar committed:
Removes commented import

The test for that component didn't actually require that component.

kognate reviewed (approved):
LGTM
